.github/workflows/tests.yml

```diff
@@ -43,12 +38,16 @@ jobs:
       with:
         go-version: ${{ matrix.go-version }}
     - uses: actions/checkout@v2
-
+    
+    # Get values for cache paths to be used in later steps 
+    - id: go-cache
+      run: echo "::set-output name=go-build::$(go env GOCACHE)"
```

Comment:
might as well use this approach for `go env GOMODCACHE` too?  I mean, it's not likely to change, but you never know and the consistency is nice.

Reply:
I agree